importer/management/commands/dedupe_categories.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from cms.categories.models import CategoryPageCategoryRelationship, Category
+from cms.publications.models import PublicationType
+
+logger = logging.getLogger(__name__)
+
+DELETE = []
+# from: to
+MERGE = {
+    "Allied health professionals bulletin": "Allied Health Professionals",
+    "Business plan 2012-2015: indicators and other key data": "Business plan",
+    "CCG learning network news": "CCG learning and support tool",
+    "Commissioning Support Bulletin": "Commissioning",
+    "Diabetes": "Diabetes and Kidney Care",
```

Comment:
These feel like different things, might be worth double-checking this one in particular.